seqr/views/apis/project_api.py

```diff
@@ -184,34 +184,46 @@ def project_page_data(request, project_guid):
 @login_and_policies_required
 def project_families(request, project_guid):
     project = get_project_and_check_permissions(project_guid, request.user)
-    family_models = Family.objects.filter(project=project).annotate(
-        metadata_individual_count=Count('individual', filter=Q(
-            individual__features__0__isnull=False, individual__birth_year__isnull=False,
-            individual__population__isnull=False, individual__proband_relationship__isnull=False,
-        )),
-        pp_individual_count=Count('individual', filter=Q(
-            individual__phenotypeprioritization__tool__isnull=False,
-        ))
-    )
+
+    family_models = Family.objects.filter(project=project)
     families = family_models.values(
-        'description',
+        'id', 'description',
         **{_to_camel_case(field): F(field) for field in [
             'family_id', 'analysis_status', 'created_date', 'coded_phenotype', 'mondo_id',
         ]},
         familyGuid=F('guid'),
         projectGuid=Value(project_guid),
         **FAMILY_ADDITIONAL_VALUES,
-        **INDIVIDUAL_GUIDS_VALUES,
```

Comment:
the join that was being constructed for all of these annotations was far less efficient than running 3 queries directly - on on families, one on individuals, and one on phenotype prioirty. The `parents` annoattons was causing us to do 3 separate joins on the individuals table, and the phenotype priority data is relatively sparse so it turned out the full outer join on the individual tabel is slower than the very targeted aggregation directly on that table